src/main/web/templates/handlebars/partials/header.handlebars

```diff
@@ -113,6 +103,15 @@
 						</a>
 					</li>
 				</ul>
+                <div class="language hide--md primary-nav__link">
+                    {{#if_eq location.hostname (concat "cy." (replace location.hostname "cy." ""))}}
+                        <a href="{{> partials/language-toggle language="en"}}" class="language__link__mb">English (EN)</a>
```

Comment:
Similar to my comment on the Sixteens pull request - rather than using a specific `languague__link__mb` class we should probably be using `language__link` class but adding a media query into CSS.